core/io/resource.h

```diff
@@ -39,11 +39,15 @@
 
 class Node;
 
-#define RES_BASE_EXTENSION(m_ext)                                                                                   \
-public:                                                                                                             \
-	static void register_custom_data_to_otdb() { ClassDB::add_resource_base_extension(m_ext, get_class_static()); } \
-	virtual String get_base_extension() const override { return m_ext; }                                            \
-                                                                                                                    \
+#define RES_BASE_EXTENSION(m_ext)                                        \
```

Comment:
My visual studio is religiously following the clang format so it formats the existing code whenever I save `resource.h`

Reply:
What is your clang-format version? IIRC there was some bug in the newest one (14 I think), so maybe this is the reason.

Reply:
I downloaded Clang from Visual

Reply:
You must have been running 14.0.0 because that's the version which does those unnecessary changes. They would need to be reverted.

Reply:
That's weird, even when I explicitly bind my clang format to the v12.0.0 exe, it still reformats the entire resource.h file. What version do you use?

Reply:
Also, I notice I'm getting the error "Unknown key: SpacesInLineCommentPrefix" whenever my clang-format runs.